.bumpversion.toml

```diff
@@ -0,0 +1,34 @@
+# SPDX-FileCopyrightText: : 2024 Ludwig Hülk <https://github.com/Ludee> © Reiner Lemoine Institut
+# SPDX-FileCopyrightText: super-repo v0.5.0 <https://github.com/rl-institut/super-repo>
+# SPDX-License-Identifier: MIT
+
+[tool.bumpversion]
+current_version = "0.5.0"
```

Comment:
what is this version about again?

Reply:
This is a copy fail. It should be the OEM version 2.0.2. Thanks